### PR TITLE
Replace freenode references with libera chat

### PR DESCRIPTION
**Pull request by purna135**
Signed-off-by: Purna Chandra Mansingh <purnachandramansingh135@gmail.com>

Fixes [Replace references to freenode #92](https://github.com/nexB/aboutcode/issues/92)

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

**Comment by pombredanne** [comment]
You wrote:

> Fixes #92

I would doubt that you meant https://github.com/nexB/scancode-toolkit/issues/92 ?
Could you instead use the full URL to the main issue in AboutCode?

File: tests/licensedcode/data/datadriven/external/fossology-tests/Apache/etcd.txt [hunk 1]
```diff
@@ -120,7 +120,8 @@ Now it's time to dig into the full etcd API and other guides.
 ## Contact
 
 - Mailing list: [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev)
-- IRC: #[etcd](irc://irc.freenode.org:6667/#etcd) on freenode.org
```
Comment (pombredanne) [inline tests/licensedcode/data/datadriven/external/fossology-tests/Apache/etcd.txt]: This is a test file... not a doc file for our code... do not modifiy this

File: tests/packagedcode/data/pypi/setup.py-versions/cffi-1.14.0/setup.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def no_working_compiler_found():
     tries to compile C code.  (Hints: on OS/X 10.8, for errors about
     -mno-fused-madd see http://stackoverflow.com/questions/22313407/
     Otherwise, see https://wiki.python.org/moin/CompLangPython or
-    the IRC channel #python on irc.freenode.net.)
+    the IRC channel https://gitter.im/aboutcode-org/discuss)
```
Comment (pombredanne) [inline tests/packagedcode/data/pypi/setup.py-versions/cffi-1.14.0/setup.py]: This is a test file... not a doc file for our code... do not modifiy this

File: tests/packagedcode/data/pypi/unpacked_wheel/metadata-2.1/pip-20.2.2.dist-info-expected.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
...
...of Conduct\n---------------\n\nEveryone interacting in the pip project's codebases, issue trackers, chat\nrooms, and mailing lists is expected to follow the `PyPA Code of Conduct`_.\n\n.. _package installer: https://packaging.python.org/guides/tool-recommendations/\n.. _Python Package Index: https://pypi.org\n.. _Installation: https://pip.pypa.io/en/stable/installing.html\n.. _Usage: https://pip.pypa.io/en/stable/\n.. _Release notes: https://pip.pypa.io/en/stable/news.html\n.. _Release process: https://pip.pypa.io/en/latest/development/release-process/\n.. _GitHub page: https://github.com/pypa/pip\n.. _Development documentation: https://pip.pypa.io/en/latest/development\n.. _learn more and take our survey: https://pyfound.blogspot.com/2020/03/new-pip-resolver-to-roll-out-this-year.html\n.. _Issue tracking: https://github.com/pypa/pip/issues\n.. _Discourse channel: https://discuss.python.org/c/packaging\n.. _Development mailing list: https://mail.python.org/mailman3/lists/distutils-sig.python.org/\n.. _User IRC: https://gitter.im/aboutcode-org/discuss\n.. _Development IRC: https://web.libera.chat/?#aboutcode\n.. _PyPA Code of Conduct: https://www.pypa.io/en/latest/code-of-conduct/",
```
Comment (pombredanne) [inline tests/packagedcode/data/pypi/unpacked_wheel/metadata-2.1/pip-20.2.2.dist-info-expected.json]: This is a test file... not a doc file for our code... do not modifiy this

File: tests/packagedcode/data/pypi/unpacked_wheel/metadata-2.1/pip-20.2.2.dist-info/METADATA [hunk 4]
```diff
@@ -81,8 +81,8 @@ rooms, and mailing lists is expected to follow the `PyPA Code of Conduct`_.
 .. _Issue tracking: https://github.com/pypa/pip/issues
 .. _Discourse channel: https://discuss.python.org/c/packaging
 .. _Development mailing list: https://mail.python.org/mailman3/lists/distutils-sig.python.org/
-.. _User IRC: https://webchat.freenode.net/?channels=%23pypa
-.. _Development IRC: https://webchat.freenode.net/?channels=%23pypa-dev
+.. _User IRC: https://gitter.im/aboutcode-org/discuss
+.. _Development IRC: https://web.libera.chat/?#aboutcode
```
Comment (pombredanne) [inline tests/packagedcode/data/pypi/unpacked_wheel/metadata-2.1/pip-20.2.2.dist-info/METADATA]: This is a test file... not a doc file for our code... do not modifiy this

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks ... See my comments inline

**Comment by purna135** [comment]
Thank you so much @pombredanne, I thought I have to change every Freenode reference but I have to change the Freenode references which are only present in the documentation. 
I will do the necessary changes and will push it again.

**Review by pombredanne — APPROVED** [review]
Looking great. Thanks

**Pull request merged**